src/about-us/who-we-are.md

```diff
@@ -11,4 +11,22 @@ Our approach is to add value early and often, rapidly researching, testing, lear
 
 We’re researchers, designers, developers, delivery leads, business analysts, strategists and product and performance specialists. We work in multidisciplinary teams because bringing a diverse mix of skills, expertise and perspectives together leads to better results, and means we can respond quickly to what we’re learning about people’s needs. As well as being digital specialists, we’re also experienced coaches and mentors in the technology sector, and some of our staff hold roles outside of dxw as Charity Board Chairs, Trustees and Lecturers.
 
-Because we’re employee owned we have the freedom to work on projects we're passionate about, and where we know we can add real value. And we’re unique in that we run the company in the best interests of the staff and people who use public services (this is written into our founding documents).
+Because we’re employee owned we have the freedom to work on projects we're passionate about, and where we know we can add real value. And we’re unique in that we run the company in the best interests of the staff and people who use public services (this is written into our founding documents). 
+
+## Our teams
+
+Sales: generates a pipeline of work that’s on mission and provides work for the people we have.
+
+Delivery+: specialises in creating bespoke digital transactional services by providing multidisciplinary teams
+
+GovPress: designs, builds, hosts and supports content-first websites for public and third sector clients
+
+Strategy: helps organisations form strategies, plans and business cases, and build the capability needed for digital transformation
+
+Product and Performance 
```

Comment:
Have we missed some text here?

Reply:
(I am tempted to do some squashing here but I really don't have the dxw time!)